app.js

```diff
@@ -1,10 +1,10 @@
 function changeColor() {
     const p = document.getElementById("text");
 
-    if (p.style.color != "green") {
```

Comment:
Can we improve code readability by using this method 
`p.classList.contains("green")` which returns boolean ?